### PR TITLE
Add DELETE API to Microsoft.Security applicationWhitelistings

### PR DESCRIPTION
**Pull request by erelh**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [x] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Comment by erelh** [comment]
@ArcturusZhang @tjprescott  can you please review? thanks

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/applicationWhitelistings.json
Comment (ArcturusZhang) [inline specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/applicationWhitelistings.json]: Should it be 204 when we are trying to delete an item that does not exist?
Randomly picked example: https://github.com/Azure/azure-rest-api-specs/blob/114c018edda56641cf2615125c3dd8a397f875d3/specification/appplatform/resource-manager/Microsoft.AppPlatform/preview/2019-05-01-preview/appplatform.json#L143

**Comment by tjprescott** [comment]
LGTM. Is this PR sufficient or do you still need the other one open with all of the other changes?

**Comment by erelh** [comment]
@tjprescott i don't need the other one open, you can close it

File: specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/applicationWhitelistings.json
Comment (erelh) [reply specification/security/resource-manager/Microsoft.Security/preview/2015-06-01-preview/applicationWhitelistings.json]: @ArcturusZhang  done

**Comment by tjprescott** [comment]
Is this a management or data plane swagger? Data plane swaggers aren't reviewed by ARM. @ArcturusZhang

**Comment by ArcturusZhang** [comment]
> Is this a management or data plane swagger? Data plane swaggers aren't reviewed by ARM. @ArcturusZhang

Well, the changed file is under `resource-manager` directory, I suppose this is a management-plane swagger.

**Comment by erelh** [comment]
@ArcturusZhang @tjprescott what is blocking this PR from being reviewed & merged ?

**Comment by tjprescott** [comment]
As this is a management plane change, it should be reviewed by the ARM team first and then the management SDKs team in Shanghai. I'm part of the data plane SDKs team, so the bot should not have assigned the original PR to me. I suspect it did so because of all the changes that were pulled into that (the reason so many people were tagged as code owners). @KrisBash who would be the right ARM team member to review this?

**Comment by erelh** [comment]
@KrisBash @ArcturusZhang can you please assist in getting a reviewer for this change ?
Thanks

**Comment by erelh** [comment]
@ravbhatnagar - are you OnCall now ? can you please review?

**Review by ryansbenson — COMMENTED** [review]
Approved from ARMs side

**Review by ArcturusZhang — APPROVED** [review]

**Pull request merged**
